between random: 174430936
between random: 174430936
between random: 174430935
between random: 174430936
between random: 174430936
between random: 50287
between random: [130,3,120,54,139,152,78,22]
between random: [130,3,120,54,139,152,78,22]
between random: [130,3,120,54,139,152,78,22]
between random: [130,3,120,54,139,152,78,22]
between random: [162,120,53,23,173,176,10,2]
between random: [148,17,123,216,181,39,199,49]
sed: can't read /usr/local/lib/python2.7/dist-packages/ivy/include/1.7/quic_frame.ivy: No such file or directory
""
/usr/local/lib/python2.7/dist-packages/ivy/include/1.7/quic_frame.ivy: line 597: error: assumption failed
output directory: temp/1856
implementation directory: /home/chris/TVOQE_UPGRADE_27/quic/aioquic/
implementation command: python3 examples/http3_server.py --quic-log /home/chris/qlog_aiquic2 --certificate /home/chris/TVOQE_UPGRADE_27/quic/aioquic/tests/ssl_cert.pem --private-key /home/chris/TVOQE_UPGRADE_27/quic/aioquic/tests/ssl_key.pem  -v --host 127.0.0.1 --port 4443 -l /home/chris/secret.log
../quic_server_test_newcoid_rtp_error (0) ...
implementation command: ['python3', 'examples/http3_server.py', '--quic-log', '/home/chris/qlog_aiquic2', '--certificate', '/home/chris/TVOQE_UPGRADE_27/quic/aioquic/tests/ssl_cert.pem', '--private-key', '/home/chris/TVOQE_UPGRADE_27/quic/aioquic/tests/ssl_key.pem', '-v', '--host', '127.0.0.1', '--port', '4443', '-l', '/home/chris/secret.log']
quic_process pid: 24193
timeout 100 ./build/quic_server_test_newcoid_rtp_error seed=586 the_cid=0 server_cid=1 client_port=4987 client_port_alt=4988
client return code: 1
FAIL
error: 1 tests(s) failed
